frontend/src/store/course/index.js

```diff
@@ -26,6 +25,13 @@ const state = {
 
 const getters = {
   course: ({course}) => course,
+  ratings: ({course}) => [
+    {text: 'Recommended', value: course.recommend || 0},
```

Comment:
Should null values resolve. I guess this safe, but it also creates a problem where courses get 0 ratings if unrated, which I'm not sure on.

Reply:
Yeah ... maybe a greyed out circle with no number or 'unrated' text would be better. Feel free to add if you see a good way to do it... i'm outta time today